Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -46,9 +46,34 @@ class SettingsViewController: UIViewController, ToolbarViewProtocol, UITableView
         signOutButton.addTarget(self, action: "didClickLogout", forControlEvents: UIControlEvents.TouchUpInside)
```

Comment:
Delegate methods use `didFoo`, `shouldFoo`, `willFoo` while actions (Selectors) should use `SELclickLogout` style names. Easier to see what is what then. (I don't know where the `SELfoo` naming came from but @thebnich keeps bugging me about it - Brian, was this in the Ray Wenderlich coding style?)